ui/mainwindow.cpp

```diff
@@ -2949,7 +2949,12 @@ void MainWindow::showEvent(QShowEvent *e)
 }
 
 void MainWindow::startAutoSave() {
-    QTimer *timer = new QTimer(this);
-    connect(timer, SIGNAL(timeout()), &this->control, SLOT(saveProject()));
-    timer->start(1000 * 60 * 10); // 10 minutes
+    int i = ProjectConfig::getAutoSaveInterval();
+    qDebug() << "auto save interval: " << i << "minutes" << (i>0 ? "" : ": disabled");
```

Comment:
dient das der Protokollierung beim Start?

Reply:
Hier wird nur auf stdout protokolliert, nicht in das Console-Fenster von OI.
Wird später bei Releases deaktiviert.